platform/src/abstractions/EducationBenchmarking.Platform.Domain/DataObjects/SchoolRatingDataObject.cs

```diff
@@ -1,18 +1,19 @@
 using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
 
 namespace EducationBenchmarking.Platform.Domain.DataObjects;
 
 [ExcludeFromCodeCoverage]
-public class SchoolRatingDataObject
+public record SchoolRatingDataObject
 {
-    public string Term { get; set; }
-    public string OverallPhase { get; set; }
-    public string AssessmentArea { get; set; }
-    public string Divisor { get; set; }
-    public string Size { get; set; }
-    public string FSM { get; set; }
+    public string? Term { get; set; }
+    public string? OverallPhase { get; set; }
+    public string? AssessmentArea { get; set; }
+    public string? Divisor { get; set; }
+    public string? Size { get; set; }
+    [JsonProperty("FSM")] public string? Fsm { get; set; }
```

Comment:
Some of the other objects in this folder use constants for these attributes. Is it worth rolling this out everywhere?

Reply:
I'm hoping we won't need them when we produce our own data pipeline, and we'll follow standard JSON variable naming conventions.